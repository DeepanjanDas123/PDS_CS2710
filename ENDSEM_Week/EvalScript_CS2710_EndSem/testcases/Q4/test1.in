GenerateLengthLimitedHuffman
drjrjsbrzgzobr
4
Quit
