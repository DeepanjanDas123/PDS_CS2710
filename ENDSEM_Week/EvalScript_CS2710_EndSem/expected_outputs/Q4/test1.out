r 00
b 010
d 011
j 100
s 101
z 110
g 1110
o 1111
cost: 40
